### ISSUE TITLE
Lattice 0.4.3 doesn't pass tests on AWS

### ISSUE THREAD

**Issue by lexsys27** [cloudfoundry-incubator/lattice#197]
Hi!

I have just upgraded to Lattice 0.4.3 and can't push apps. `ltc test -v` fails with error:

```
[ERROR]   09/12 17:56:18.36 4.46.2.1.2     rep.depot-client.run-container.run.setup.download-step.fetch-failed
[debug] Download failed: Status code 404
[debug] {"cacheKey":"","guid":"lattice-app-0af63bd1-466b-4367-6bac-cf637bb8440f","to":"/tmp","user":"vcap"}
```

**Comment by davidwadden** [comment]
Just making sure, are you downloading the bundles listed on our [GitHub releases](https://github.com/cloudfoundry-incubator/lattice/releases/tag/v0.4.3) page?

Can you `ssh -i <key-file> ubuntu@<target_ip>` and run `cat /var/lattice/LATTICE_VERSION`  ?  

Or maybe simpler `tar xOf lattice.tgz lattice-build/common/LATTICE_VERSION` from the directory you ran `terraform apply`  (`lattice-bundle-v0.4.3-osx/terraform/aws`)

**Comment by lexsys27** [comment]
No, I have changed `source` and `lattice_tar_source` in `lattice.tf` file for the release 0.4.2, destroyed and then recreated env. To be clear I will repeat installation from the bundle.

Will write back in a couple of minutes.

**Comment by lexsys27** [comment]
BTW, it is a bit confusing for me that I can't just checkout git repo and pull changes to the terraform file as the version updates.

**Comment by lexsys27** [comment]
With the fresh install from bundle problem is not presented. So the issue was with partial upgrade from the previous version.

**Comment by davidwadden** [comment]
Okay, we implemented a couple stories [[#101716496](https://www.pivotaltracker.com/story/show/101716496)] [[#101716526](https://www.pivotaltracker.com/story/show/101716526)] since others had reported a similar experience of not following the new bundle approach.  After **v0.4.4**, users should get a warning when attempting to `vagrant up` from the git repo with an "old" `lattice.tgz` in the folder;  `IGNORE_VERSION_MISMATCH=true vagrant up` can be run in order to force a mismatched build to start.

We've considered the upgrade issue with the terraform configs.  It's annoying to have to cut & paste, but we have to be sensitive to keeping our terraform configs in sync with the lattice builds.  We want to enforce people to keep those two in lock step to avoid compatibility issues.

(sorry for the delay getting back)
